Minor DE Fixes (#629)
AlexsLemonade/training-modules: scRNA-seq-advanced/04-differential_expression.Rmd

The samples which could be further classified have a mix of `Tumor_Mesoderm`, `Tumor_Myoblast`, and `Tumor_Myocyte`.

```{r celltype UMAP}

> # UMAP of all libraries and labeling cell type

Reviewer: Maybe?
```suggestion
# UMAP of all samples labeled by cell type
```
